bittensor/subtensor.py

```diff
@@ -3024,189 +3029,320 @@ def validator_logits_divergence(
+            block (Optional[int], optional): The block number to retrieve the parameter from. If ``None``, the latest
+                block is used. Default is ``None``.
+
+        Returns:
+            Optional[int]: The value of the ValidatorEpochsPerReset hyperparameter, or ``None`` if the subnetwork does
+                not exist or the parameter is not found.
+        """
+        call = self._get_hyperparameter(
+            param_name="ValidatorEpochsPerReset", netuid=netuid, block=block
+        )
+        return None if call is None else int(call)
 
     def validator_epoch_length(
         self, netuid: int, block: Optional[int] = None
     ) -> Optional[int]:
-        """Returns network ValidatorEpochLen hyper parameter"""
-        if not self.subnet_exists(netuid, block):
-            return None
-        _result = self.query_subtensor("ValidatorEpochLen", block, [netuid])
-        if not hasattr(_result, "value") or _result is None:
-            return None
+        """
+        Returns network ValidatorEpochLen hyperparameter.
 
-        return _result.value
+        Args:
+            netuid (int): The unique identifier of the subnetwork.
+            block (Optional[int], optional): The block number to retrieve the parameter from. If ``None``, the latest
+                block is used. Default is ``None``.
+
+        Returns:
+            Optional[int]: The value of the ValidatorEpochLen hyperparameter, or ``None`` if the subnetwork does not
+                exist or the parameter is not found.
+        """
+        call = self._get_hyperparameter(
+            param_name="ValidatorEpochLen", netuid=netuid, block=block
+        )
+        return None if call is None else int(call)
 
     def validator_exclude_quantile(
         self, netuid: int, block: Optional[int] = None
     ) -> Optional[float]:
-        """Returns network ValidatorEpochLen hyper parameter"""
-        if not self.subnet_exists(netuid, block):
-            return None
-        _result = self.query_subtensor("ValidatorExcludeQuantile", block, [netuid])
-        if not hasattr(_result, "value") or _result is None:
-            return None
-        return U16_NORMALIZED_FLOAT(_result.value)
+        """
+        Returns network ValidatorExcludeQuantile hyperparameter.
+
+        Args:
+            netuid (int): The unique identifier of the subnetwork.
+            block (Optional[int], optional): The block number to retrieve the parameter from. If ``None``, the latest
+                block is used. Default is ``None``.
+
+        Returns:
+            Optional[float]: The value of the ValidatorExcludeQuantile hyperparameter, or ``None`` if the subnetwork
+                does not exist or the parameter is not found.
+        """
+        call = self._get_hyperparameter(
+            param_name="ValidatorExcludeQuantile", netuid=netuid, block=block
+        )
+        return None if call is None else float(call)
```

Comment:
Should this `float(call)` be using  `U16_NORMALIZED_FLOAT` instead?

Reply:
Good catch, Abe! Thanks!